CMakeLists.txt

```diff
@@ -275,7 +275,7 @@ if(GCC OR CLANG)
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fvisibility=hidden -Wall -Wextra -Wno-unused-parameter -Werror")
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wunused -Wcomment -Wchar-subscripts -Wuninitialized -Wshadow")
     set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wwrite-strings -Wformat-security -Wunused-result")
-    set(C_CXX_FLAGS "${C_CXX_FLAGS} -Wvla")
+    set(C_CXX_FLAGS "${C_CXX_FLAGS} -Wvla -Wtype-limits")
```

Comment:
`CMAKE_C_FLAGS` only applies to C while `C_CXX_FLAGS` is applied to C and C++ (see lines 333 and 334). This looks like it will be redundant for C builds but it won't hurt anything